### PR TITLE
Added wager, greatest game in all of fatherland

### PR DESCRIPTION
**Pull request by makoConstruct**
Now, predicting the future is a game we can play.

Note. None of this is in any way tested. Havn't looked at the test API yet. Need to sleep.

### CODE REVIEW

File: module.goat.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ def tower
             "The goat laughs \"BAAAHHHHHHHHHHAHAHA\" as you are forced to leap from the goat tower",
             "The scroll from the elder goat states there is a passage way with a green star engraved. You might become rich if you can find this passage.",
             "The poster reads: \"There is no futBAAAAAAHHHHure but what we mBAAAAHHHHHHHHke for ouBAAAAAAHHHHHHHHHHHHHrselves.\""
```
Comment (passcod) [inline module.goat.rb]: You need a comma there.

File: module.wager.rb [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+		end
+		#ensure that the bet doesn't already exist
+		nope = false
+		betcall = "wager #{name}"
+		for triple in @memory.load(betcall).each
+			if triple[0] == 'ludivine' and triple[1] == betcall
+				nope = true
+				break
+			end
+		end
```
Comment (passcod) [inline module.wager.rb]: That can be rewritten in idiomatic ruby as

``` ruby
if @memory.load(betcall).find{|triple| triple[0] == 'ludivine' and triple[1] == betcall}
  wdescription = ...
end

```

File: module.wager.rb [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+		#fetch bet information
+		for triple in @memory.load(betcall)
+			if triple[0] == 'ludivine' and triple[1] == betcall
+				splitted = triple[2].split('::')
+				description = splitted[0]
+				options = splitted[1..splitted.length]
+				break
+			end
+		end
```
Comment (passcod) [inline module.wager.rb]: Same here:

``` ruby
bet = @memory.load(betcall).find{|triple| triple[0] == 'ludivine' and triple[1] == betcall}
return "There is no such wager as #{name}" unless bet
options = triple[2].split('::')
description = options.shift

wagerBetQ = ...
```

etc

**Comment by makoConstruct** [comment]
Too many API and structural changes

**Pull request closed without merging**